doc/INSTALLATION.md

```
@@ -11,6 +11,7 @@
 - [安装方式二：下载打包版本](#安装方式二下载打包版本)
 - [安装方式三：从源码运行](#安装方式三从源码运行)
 - [安装方式四：Docker部署](#安装方式四docker部署)
+- [安装方式五：macOS 原生运行（Apple Silicon）](#安装方式五macos原生运行apple-silicon)
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)